src/manager.rs

```diff
@@ -26,7 +26,7 @@ pub struct JoinOptions {
 
 #[derive(Debug)]
 pub struct GameWrapper {
-    game: Game,
+    pub game: Game,
```

Comment:
We should try and keep `game` private, it's the last bastion of private state in this dang thing, everything else can be changed by anything really. Make a method instead that gets the `game_created` time.